src/components/LocationSelector/LocationSelector.tsx

```diff
@@ -213,7 +213,7 @@ export const LocationSelector: React.FC<Props> = (props) => {
                 isFocused={isWrapperFocused}
                 showClearButton={hasLocationsSelected}
                 clearLabel={clearLabel}
-                onClick={handleOpenModal}
+                onMouseUp={handleOpenModal}
```

Comment:
I was investigating why you had to use `any` for `LocationSelector`...  Although, the reason is different, could you pls fix these types:

* [onAddLocation](https://github.com/textkernel/oneui/pull/299/files#diff-8ecbfeea0d57301b84c682e7f6873a11R72)
* [onUpdateLocation](https://github.com/textkernel/oneui/pull/299/files#diff-8ecbfeea0d57301b84c682e7f6873a11R74)
* [onRemoveLocation](https://github.com/textkernel/oneui/pull/299/files#diff-8ecbfeea0d57301b84c682e7f6873a11R74)

Should be like this:
```typescript
onAddLocation: (location: Location) => void;
onUpdateLocation: (location: Location) => void;
onRemoveLocation: (location: Location) => void;
```